src/index.js

```diff
@@ -1,27 +1,61 @@
 /**
  * @flow
  */
+import * as emails from './store/emails.json';
```

Comment:
With your Node updated, you should be able to change this now to `import store from './store';`

Reply:
Yep! Got it working now. For me it actually had to do with how I was writing the import statement. I was writing {store} instead of just store, which I learned isn't necessary since it's the default export